modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+Run the following command to check for existing {istio} instances.
+[source,terminal]
+----
+$ oc get istios
+----
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Add the `istio-discovery=mesh-1` label to ensure that the namespace is discovered by the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio-discovery=mesh-1
+----
+
+. Add the `istio.io/rev=mesh-1` label to the namespace to enable sidecar injection by default while ensuring that pods in this namespace are mapped to the first control plane.
```

Comment:
Is it clear what "by default" means? Maybe clarify by saying "... to enable sidecar injection into _all pods_ by default", since that's what this label does (i.e. without this namespace label, users would have to enable injection for each pod individually).